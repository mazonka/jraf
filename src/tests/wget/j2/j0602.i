command=jw md 0 /aaa : save 0 /aaa/c1 10 MTIzNDU2Nzg5MA== : rm 0 /aaa/c1 : get 0 /