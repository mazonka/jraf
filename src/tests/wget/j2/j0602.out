OK /aaa : OK 10 : OK /aaa/c1 : OK 3 -1
==
